packages/math-input/src/components/input/mathquill-types.ts

```diff
@@ -3,73 +3,339 @@ import Key from "../../data/keys";
+     */
     clearSelection: () => void;
-    // Custom helper in our MathQuill fork
-    // check to see if cursor is on right end
+
+    /**
+     * Move the cursor to the left end of the editable field. horthand
+     * for .moveToDirEnd(L)
+     */
+    moveToLeftEnd: () => void;
+
+    /**
+     * Move the cursor to the right end of the editable field. horthand
+     * for .moveToDirEnd(R)
+     */
+    moveToRightEnd: () => void;
+
+    /**
+     * Moves the cursor to the end of the mathfield in the direction
+     * specified. The direction can be one of MQ.L or MQ.R. These are
+     * constants, where MQ.L === -MQ.R and vice versa. This function
+     * may be easier to use than moveToLeftEnd or moveToRightEnd if
+     * used in the moveOutOf handler.
+     *
+     * https://docs.mathquill.com/en/latest/Api_Methods/#movetodirenddirection
+     */
+    moveToDirEnd: (direction: MathQuillDirection) => void;
+
+    /**
+     * Simulates keystrokes given a string like "Ctrl-Home Del", a
+     * whitespace-delimited list of key inputs with optional prefixes.
+     *
+     * ```
+     * mathField.keystroke('Shift-Left'); // Selects character before the current cursor position
+     * ```
+     *
+     * https://docs.mathquill.com/en/latest/Api_Methods/#keystrokekeys
+     */
+    keystroke: (keys: string) => MathFieldInterface;
+
+    /**
+     * Simulates typing text, one character at a time from where the
+     * cursor currently is. This is supposed to be identical to what
+     * would happen if a user were typing the text in.
+     *
+     * Types part of the demo from mathquill.com without delays between keystrokes
+     * ```
+     * mathField.typedText('x=-b\\pm \\sqrt b^2 -4ac');
+     * ```
+     *
+     * https://docs.mathquill.com/en/latest/Api_Methods/#typedtexttext
+     */
+    typedText: (text: string) => MathFieldInterface;
+
+    /**
+     * When called withot any pramaters, gets the contents as LaTeX
+     * When passed a string, sets the contents as LaTeX
+     *
+     * https://docs.mathquill.com/en/latest/Api_Methods/#latex
+     */
+    latex: (input?: string) => string;
+
+    /** Changes the configuration of just this math field. */
+    config: (new_config: MathFieldConfig) => void;
+
+    /**
+     * KA Custom helper in our MathQuill fork check to see if cursor is on
+     * right end
+     */
     cursorAtRightEnd: () => boolean;
-    // Custom helper in our MathQuill fork
-    // check to see if cursor is on left end
+
+    /**
+     * KA Custom helper in our MathQuill fork check to see if cursor is on left
+     * end
+     */
     cursorAtLeftEnd: () => boolean;
-    // This isn't part of the MathQuill public API
-    // I don't know what it is and it feels wrong using it
+
+    /**
+     * This isn't part of the MathQuill public API
+     * I don't know what it is and it feels wrong using it
+     *
+     * @deprecated This is internal and shouldn't be used.
```

Comment:
This, specifically, was the part that VSCode wouldn't recognize unless it was in these multi-line comments (`/** */`) vs single-line style (`// ...`)